README.md

```diff
@@ -67,7 +67,7 @@ RustScans **only** job is to reduce the friction between finding open ports and
 
 | Installation Guide | Documentation | Discord |
 | ------------------ | ------------- | ------- |
-| 📖 [Installation Guide](https://github.com/RustScan/RustScan#-full-installation-guide) | 📚 [Documentation](https://github.com/RustScan/RustScan/issues/89) | 🦜 [Discord](https://discord.gg/GFrQsGy)
+| 📖 [Installation Guide](https://github.com/RustScan/RustScan#-full-installation-guide) | 📚 [Documentation](http://rustscan.github.io/RustScan) | 🦜 [Discord](https://discord.gg/GFrQsGy)
```

Comment:
```suggestion
| 📖 [Installation Guide](https://github.com/RustScan/RustScan#-full-installation-guide) | 📚 [Documentation](https://rustscan.github.io/RustScan/) | 🦜 [Discord](https://discord.gg/GFrQsGy)
```

Reply:
Thanks @SuperSandro2000. The [GitHub Pages guide's](https://pages.github.com/) example for project pages uses HTTP and not HTTPS, but after double checking the redirect to HTTPS is automatic, so there is no reason indeed to not use it.